mlrun/runtimes/databricks_runtime_wrap.py

```diff
@@ -0,0 +1,173 @@
+):
+    import ast
+    import datetime
+    import json
+    import os
+    import shutil
+    import tempfile
+    import uuid
+
+    import astor
+    from databricks.sdk import WorkspaceClient
+    from databricks.sdk.service.compute import ClusterSpec
+    from databricks.sdk.service.jobs import Run, SparkPythonTask, SubmitTask
+
+    import mlrun
+
+    project_name = context.project
+    project = mlrun.get_or_create_project(project_name)
+    get_secret_func = project.get_secret
+    logger = context.logger
+
+    def get_modified_code(function_name, script_file, handler):
+        with open(script_file, "r") as file:
+            tree = ast.parse(file.read())
+
+        # Find the function node to delete
+        to_delete = None
+        for node in tree.body:
+            if isinstance(node, ast.FunctionDef) and node.name == function_name:
+                to_delete = node
+                break
+
+        if to_delete:
+            # Remove the function node
+            tree.body.remove(to_delete)
+
+        # Generate the modified code
+        modified_code = astor.to_source(tree)
+        if handler:
+            handler_function = f"""
+import argparse
+import json
+parser = argparse.ArgumentParser()
+parser.add_argument('handler_arguments', help='')
+handler_arguments = parser.parse_args().handler_arguments
+handler_arguments = json.loads(handler_arguments)
+{handler}(**handler_arguments)
+"""
+            modified_code += handler_function
+        return modified_code
+
+    def copy_current_file(dst_dir):
+        current_script = os.path.abspath(__file__)
+        # Create a temporary directory and copy the file inside the context
+        # Generate the temporary file path
+        temp_file_path = os.path.join(dst_dir, os.path.basename(current_script))
+        # Copy the current script to the temporary location
+        shutil.copy(current_script, temp_file_path)
+        return temp_file_path
+
+    def upload_file(workspace: WorkspaceClient, script_path_on_dbfs: str, handler):
+        with tempfile.TemporaryDirectory(
+            prefix="databricks_runtime_scripts_"
+        ) as temp_dir:
+            temp_file_path = copy_current_file(temp_dir)
+            modified_code = get_modified_code(
+                function_name="run_mlrun_databricks_job",
+                script_file=temp_file_path,
+                handler=handler,
+            )
+            with workspace.dbfs.open(
+                script_path_on_dbfs, write=True, overwrite=True
+            ) as f:
+                f.write(modified_code.encode("UTF8"))
+
+    workspace = WorkspaceClient(
+        token=mlrun.get_secret_or_env(key=token_key, secret_provider=get_secret_func)
+    )
+
+    now = datetime.datetime.now()
+    formatted_date_time = now.strftime("%Y-%m-%d_%H-%M-%S")
+    script_path_on_dbfs = (
+        f"/home/{workspace.current_user.me().user_name}/mlrun_databricks_run"
+        f"time/sample_{formatted_date_time}_{uuid.uuid4()}.py"
```

Comment:
A little picky, but why are you splitting the word "runtime" in half? break it to strings in one of the `/` markers...